src/transformers/models/clip/image_processing_clip.py

```diff
@@ -101,11 +103,12 @@ def __init__(
         image_mean: Optional[Union[float, List[float]]] = None,
         image_std: Optional[Union[float, List[float]]] = None,
         do_convert_rgb: bool = True,
+        default_to_square_for_size: bool = False,
```

Comment:
```suggestion
        use_square_size: bool = False,
```
or something like this!